docs/running/server/docker-compose.yml

```diff
@@ -5,29 +5,43 @@ services:
   rabbitmq:
     image: rabbitmq:3.9.5
     hostname: rabbitmq
-    container_name: rabbitmq
-  celery-worker:
-    build:
-      context: .
-      dockerfile: Dockerfile
-    container_name: celery-worker
+  celery:
+    image: ${TOIL_APPLIANCE_SELF}
     volumes:
-      - /tmp/toil-workflows:/app/toil-workflows
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /var/lib/docker:/var/lib/docker
+      - /var/lib/toil:/var/lib/toil
+      - /var/lib/cwl:/var/lib/cwl
+      - /tmp/toil-workflows:/tmp/toil-workflows
     command: celery --broker=amqp://guest:guest@rabbitmq:5672// -A toil.server.celery_app worker --loglevel=INFO
     depends_on:
       - rabbitmq
   wes-server:
-    build:
-      context: .
-      dockerfile: Dockerfile
-    container_name: wes-server
+    image: ${TOIL_APPLIANCE_SELF}
     volumes:
-      - /tmp/toil-workflows:/app/toil-workflows
+      - /tmp/toil-workflows:/tmp/toil-workflows
     environment:
       - TOIL_WES_BROKER_URL=amqp://guest:guest@rabbitmq:5672//
-    command: toil server --host 0.0.0.0
-    ports:
-      - "127.0.0.1:8080:8080"
+    command: toil server --host 0.0.0.0 --port 8000 --work_dir /tmp/toil-workflows
+    expose:
+      - 8000
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.wes.rule=Host(`localhost`)"
```

Comment:
@Richard-Hansen Hmm, I suspect it has something to do with how traefik is configured. e.g., since `Host` is `localhost` here, requests to `localhost:8080` will be redirected to this container. (I have to update the docs to mention this as well).

Does `curl http://localhost:8080/ga4gh/wes/v1/service-info` return `404` as well?

I tried:

```
dockstore workflow wes launch \
    --entry github.com/briandoconnor/dockstore-workflow-md5sum:1.4.0 \
    --json test.json \
    --wes-url http://localhost:8080/ga4gh/wes/v1 \
    --wes-auth 'Basic dGVzdDp0ZXN0'
```
and it seemed to work (although the execution failed because I didn't include the input files).

Reply:
@w-gao It looks like the issue was on my end, after running an update of my docker daemon I'm able to hit the service info endpoint `http://localhost:8080/ga4gh/wes/v1/service-info`. Sorry for the confusion!

Since the launch was working, I ran through some tests of a few different workflows, and was able to successfully launch the example's CWL workflow, but am running into an executor error with WDL workflows. I'll create a separate issue for there to continue the discussion, as it's out-of-scope for this PR (I think at least, feel free to push back if I'm wrong).

The original issue I opened regarding WES launching errors: https://github.com/DataBiosphere/toil/issues/4026, seems to be resolved by these changes. Will leave a not on that PR and close. Thanks for the help!